0.Preprocess_Data/1.preprocessing.ipynb

```diff
@@ -6,16 +6,16 @@
    "id": "1ed18d57",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.    conditions = [
</pre><p>I think you could remove the second conditions list here. There might be another way to refactor this as well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>